### PR TITLE
[Dont' merge yet] Added tests for Storing Incomplete Responses

### PR DESCRIPTION
**Pull request by akrambek**

### CODE REVIEW

File: src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/cache.incomplete.response.with.range/accept/client.rpt [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        "nukleus://http-cache/streams/source"
+        option nukleus:route ${newProxyAcceptRef}
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${http:header(":scheme", "http")}
+                        ${http:header(":method", "GET")}
+                        ${http:header(":path", "/path")}
+                        ${http:header(":authority", "localhost:8080")}
+                        ${http:header(":content-range", "bytes 0-5/11")}
```
Comment (dpwspoon) [inline src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/cache.incomplete.response.with.range/accept/client.rpt:58]: I don't think content-range is a pseudo header

Comment (dpwspoon) [reply src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/cache.incomplete.response.with.range/accept/client.rpt:58]: I believe the request uses the Range header "Range: bytes=0-1023" and the response sends the content-range header??

File: src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/cache.incomplete.response.with.range/accept/client.rpt [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+connected
+
+write close
+
+read nukleus:begin.ext ${http:header(":status", "206 (Partial Content)")}
+                       ${http:header("content-length", "11")}
+                       ${http:header("cache-control", "max-age=15")}
+                       ${http:header("date", date)}
+                       ${http:header("last-modified", date)}
```
Comment (dpwspoon) [inline src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/cache.incomplete.response.with.range/accept/client.rpt:68]: missing content-range headers

File: src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/cache.incomplete.response.with.range/connect/server.rpt [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+write nukleus:begin.ext ${http:header(":status", "200")}
+                        ${http:header("content-length", "11")}
+                        ${http:header("cache-control", "max-age=15")}
+                        ${http:header("date", date)}
+                        ${http:header("last-modified", date)}
+
+write "hello"
+
+write close
```
Comment (dpwspoon) [inline src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/cache.incomplete.response.with.range/connect/server.rpt:46]: nit, newline here and below

**Review by dpwspoon — CHANGES_REQUESTED** [review]

**Comment by akrambek** [comment]
I am closing this PR since we are not going to implement it yet.

**Pull request closed without merging**